content/quickstart/aws/create-project.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Create a New Project
+weight: 5
+menu:
+  quickstart:
+    parent: aws
+    identifier: aws-create-project
+---
+
+Let's get started with a new project in a new directory.
+
+{{< langchoose nogo >}}
+
+<div class="language-prologue-javascript"></div>
+
+```bash
+$ mkdir quickstart && cd quickstart
+$ pulumi new aws-javascript
+```
+
+<div class="language-prologue-typescript"></div>
+
+```bash
+$ mkdir quickstart && cd quickstart
+$ pulumi new aws-typescript
+```
+
+<div class="language-prologue-python"></div>
+
+```bash
+$ mkdir quickstart && cd quickstart
+$ pulumi new aws-python
+```
+
+> If this is your first time running `pulumi new` (or most other `pulumi` commands), you will be prompted to login to the [Pulumi service](https://app.pulumi.com). The CLI works in tandem with the Pulumi service to deliver a reliable experience, and is free for individual use, with [additional features available for teams](https://www.pulumi.com/pricing/). Hitting enter at the prompt will open a web browser allowing you to sign in or sign up. Alternatively, you may [login]({{< relref "/reference/cli/pulumi_login.md" >}}) or [logout]({{< relref "/reference/cli/pulumi_logout.md" >}}) explicitly if you prefer. See the [FAQ]({{< relref "/reference/faq.md#can-i-use-pulumi-without-depending-on-pulumi-com" >}}) for more information.
```

Comment:
I have to admit, I wish this page were shorter. It's a lot to take in.

Reply:
There’s definitely room to edit down. One thing we could do is break this up into two separate steps:
 - the first being about the `new` command and prompts, and briefly introducing the project & stack concepts along the way
 - followed by another step that discusses the initial generated program.

Reply:
Split these pages into two.